swap/src/bin/swap.rs

```diff
@@ -268,6 +273,86 @@ async fn main() -> Result<()> {
 
             cli::refund(swap_id, Arc::new(bitcoin_wallet), db, force).await??;
         }
+        Command::ListMakers {
+            rendezvous_node_peer_id,
+            rendezvous_node_addr,
+            namespace,
+            tor_socks5_port,
+        } => {
+            // TODO: Fix tracing to be able to operate without swap ID
+            cli::tracing::init(debug, json, data_dir.join("logs"), Uuid::new_v4())?;
+            let seed = Seed::from_file_or_generate(data_dir.as_path())
+                .context("Failed to read in seed file")?;
+
+            let identity = seed.derive_libp2p_identity();
+
+            // TODO: Compose behaviour that allows to requests Quotes as well
+            let behaviour = list_makers::Behaviour {
+                rendezvous: Rendezvous::new(identity, rendezvous::Config::default()),
+                quote: quote::cli(),
+            };
+            let mut swarm = swarm::cli(&seed, tor_socks5_port, behaviour).await?;
+
+            let _ = swarm.dial_addr(rendezvous_node_addr.clone());
+
+            while let Some(event) = swarm.next().await {
+                match event {
+                    SwarmEvent::ConnectionEstablished { peer_id, .. }
+                        if peer_id == rendezvous_node_peer_id =>
+                    {
+                        tracing::info!(
+                            "Connected to rendezvous point, discovering nodes in '{}' namespace ...",
+                            namespace
+                        );
+
+                        swarm.behaviour_mut().rendezvous.discover(
+                            Some(Namespace::new(namespace.clone()).unwrap()),
```

Comment:
I would hardcode the namespace here.

Reply:
Maybe I misunderstand what you mean...
We can't really hardcode the namespace here, because it is blockchain network dependent. But we decide that when constructing the command already.

Reply:
I meant, that you should decide the namespace here, rather than passing it through to the command, i.e. as late as possible to make it obvious that this is not globally configurable.